django_celery_beat/migrations/0014_auto_20200609_1048.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.6 on 2020-06-09 10:48
```

Comment:
this migration needs to be newly generated after pulling from master

Reply:
Do you mind if I wait for the full review of the patch?

Reply:
https://github.com/celery/django-celery-beat/blob/master/django_celery_beat/migrations/0014_remove_clockedschedule_enabled.py this is the last working migration which serial number seems same to this, thats why I was suggesting the remove and regenration :D

Reply:
Now it's pointing to `0014_remove_clockedschedule_enabled`. Let me know if any problem persists.